.travis.yml

```diff
@@ -15,33 +23,58 @@ go:
   - tip # Go version is defined by the build container in the Dockerfile
 
 env:
-  - GORACE="halt_on_error=1"
-    SOURCE_VERSION=$(source ./build-version.txt && echo -n $SOURCE_VERSION)
-    SOURCE_REPO=$(source ./build-version.txt && echo -n $SOURCE_REPO)
-    BUILD_VERSION=$(echo ${TRAVIS_COMMIT} | cut -c 1-10)
+  global:
+    - GORACE="halt_on_error=1"
+    - BUILD_TAG=$(source ./build-version.txt && echo -n $BUILD_TAG)
+    - BUILD_COMMIT=$(source ./build-version.txt && echo -n $BUILD_COMMIT)
+    - IMAGE_NAME="iov1/tendermint:${BUILD_TAG}"
+    - BUILD_VERSION=$(echo ${TRAVIS_COMMIT} | cut -c 1-10) # random id
```

Comment:
This is not a random ID. Is this on purpose?

Reply:
ja, any unique id is good enough...